### ISSUE TITLE
Block Rule Suppress not working with max_line_length

### ISSUE THREAD

**Issue by pablisco** [pinterest/ktlint#1372]
## Expected Behavior

We have code that looks similar to this (on Android):

```kotlin
object Strings(
  val veryLongTitle = Content.String(R.string.some_module_with_long_path_title),
  val veryLongBody = Content.String(R.string.some_module_with_long_path_body),
  val veryLongDescription = Content.String(R.string.some_module_with_long_path_description),
)
```
The length in this example is shorter than it actually is in our code. It's longer than our limit of 100 there.

We were hoping to do something like this to suppress warnings:

```kotlin
object Strings(
  /* ktlint-disable max_line_length */
  val veryLongTitle = Content.String(R.string.some_module_with_long_path_title),
  val veryLongBody = Content.String(R.string.some_module_with_long_path_body),
  val veryLongDescription = Content.String(R.string.some_module_with_long_path_description),
  /* ktlint-enable max_line_length */
)
```

We would expect there no not be any warnings.

## Observed Behavior

Using the block suppress method didn't suppress the warning. 

Doing a complete disable does seem to work:

```kotlin
object Strings(
  /* ktlint-disable */
  val veryLongTitle = Content.String(R.string.some_module_with_long_path_title),
  val veryLongBody = Content.String(R.string.some_module_with_long_path_body),
  val veryLongDescription = Content.String(R.string.some_module_with_long_path_description),
  /* ktlint-enable */
)
```

But, we don't want to accidentally get something out of whack on that part of the code. 😅 

## Your Environment

* Version of ktlint used: 0.44.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): [ktlint-gradle](https://github.com/JLLeitschuh/ktlint-gradle) 10.1.0
* Version of Gradle used (if applicable): 7.4
* Operating System and version: macOS Monterey 12.1
* Link to your project (if it's a public repository): N/A

**Comment by paul-dingemans** [comment]
You are using the wrong rule id. It should be "max-line-length". Run ktlint with option "--verbose" to see the correct id.
